libvast/vast/detail/process.hpp

```diff
@@ -23,17 +23,6 @@ namespace vast::detail {
 ///             mmaped region in order to succeed.
 /// @returns The filesystem path to the library or executable mapped at address
 ///          addr, or error if the resolution fails.
-caf::expected<path> objectpath(const void* addr);
-
-/// Locates the path of a shared library or executable.
-/// @returns The filesystem path to the library or executable of the caller of
-///          this function, or error if the resolution fails.
-inline caf::expected<path> objectpath() {
-  struct dummy {
-    static void fn() {
-    }
-  };
-  return objectpath(reinterpret_cast<const void*>(&dummy::fn));
-}
```

Comment:
Just for my information: why is this function no longer needed?

Reply:
This overload is replaced through the default parameter that was added to the other overload. This was buggy because it turned out that passing an address from within the mmapped region of the executable makes `glibc` to something unexpected.